Production/crab/config/2018/DY.txt

```diff
@@ -0,0 +1,11 @@
+sampleType=MC_18 applyTriggerMatch=True saveGenTopInfo=False saveGenBosonInfo=True saveGenJetInfo=True
+
+DYJetsToLL_M-50_nlo	                /DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1/MINIAODSIM
+DYJetsToLL_M-50_nlo	                /DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v1/MINIAODSIM
```

Comment:
- duplicated name
- /DYJetsToLL_M-50_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15_ext2-v1/MINIAODSIM is missing

Reply:
- name `DYJetsToLL_M-50_nlo` does not correspond to the dataset name (madgraphMLM is LO sample)
- both NLO samples are missing